### PR TITLE
[Feature] Add a CLI feature

### PR DESCRIPTION
**Pull request by lucafaggianelli**
Allows `keepachangelog` to be used via command line:

```sh
# Run it as a Python module
python -m keepachangelog --help
# or as a shell command
keepachangelog --help
```

There are 2 main commands, `show` to print a release body and `release` to create a new release.
Here some examples of how to use the CLI:

```sh
# show
keepachangelog show 1.0.0
keepachangelog show 1.0.0 --raw
keepachangelog show 1.0.0 path/to/CHANGELOG.md

# release
keepachangelog release
keepachangelog release 1.0.1
keepachangelog release 1.0.1 -f path/to/CHANGELOG.md
```

## Motivations

The main reason behind this PR is to use `keepachange` in a CI/CD environment for creating releases in GitHub or GitLab, that is, think about running something `GITHUB_RELEASE_BODY=$(keepachangelog show ${GIT_TAG} --raw)` and creating a release via GitHub API.

The CLI is also useful for creating new releases just running `keepachangelog release`.

### CODE REVIEW

**Comment by Gustry** [comment]
Nice addition indeed ! Thanks.
I'm interested into this PR. Is there something to add on this PR ?

File: README.md [hunk 1]
```diff
@@ -299,3 +299,14 @@ Note: [flask-restx](https://pypi.python.org/pypi/flask-restx) module must be ins
 ```sh
 python -m pip install keepachangelog
 ```
+
+## Usage from command line
```
[the remark this answers is not in the document]
Comment (lucafaggianelli) [reply README.md:303]: Yeah make sense, I added the output of the `--help` to the README. Does it work for you?

File: keepachangelog/__main__.py [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+    parser_release.set_defaults(func=_command_release)
+
+    parser.add_argument(
+        "-v", "--version", action="version", version=f"%(prog)s {__version__}"
+    )
+
+    return parser.parse_args(cmdline)
+
+
+def main(cmdline=None):
```
[the remark this answers is not in the document]
Comment (lucafaggianelli) [reply keepachangelog/__main__.py]: I added the type annotation, but what would you change to make it pythonic?

**Comment by Gustry** [comment]
Thanks ! 🚀

**Comment by Colin-b** [comment]
Release 2.0.0.dev4 is now available with those changes. Thanks again for the contribution !

**Comment by lucafaggianelli** [comment]
Ah good news that’s great! ❤️

File: keepachangelog/__main__.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (Colin-b) [reply keepachangelog/__main__.py]: Renaming the parameter to use snake case instead of pascal case :)

**Pull request merged**